### PR TITLE
MODLD-596: mod-linked-data - Fix Gatling performance test

### PR DESCRIPTION
**Pull request by pkjacob**
## Purpose
Fix failing Gatling performance test of mod-linked-data

## Approach
1. Updated the login process to use 'callonce,' ensuring no repeated login attempts occur during performance testing.
2. Previously, performance tests for mod-linked-data included calls to mod-inventory and mod-search, which was unnecessary. This update isolates the performance testing to focus only on mod-linked-data calls.

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

File: mod-linked-data/src/main/resources/citation/mod-linked-data/features/suppress-flags/suppress-flags.feature [hunk 1]
```diff
@@ -5,6 +5,7 @@ Feature: Suppress Flags in Inventory
     * call login testUser
     * def testUserHeaders = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
     * configure headers = testUserHeaders
+    * configure retry = { count: 10, interval: 2000 }
```
Comment (pkjacob) [inline mod-linked-data/src/main/resources/citation/mod-linked-data/features/suppress-flags/suppress-flags.feature:8]: Without this, test could run indefinitely.

File: mod-linked-data/src/test/java/org/folio/CreateResourceSimulation.scala [hunk 2]
```diff
@@ -19,22 +19,25 @@ class CreateResourceSimulation extends Simulation {
...
   protocol.runner.systemProperty("testTenant", generateTenantId())
 
   val before = scenario("before")
     .exec(karateFeature("classpath:citation/mod-linked-data/linked-data-junit.feature"))
+  val createRefData = scenario("createRefData")
     .exec(karateFeature("classpath:citation/mod-linked-data/create-ref-data.feature"))
-  val create = scenario("create")
+  val createResource = scenario("createResource")
     .repeat(10) {
       exec(karateFeature("classpath:citation/mod-linked-data/features/create-bib-api/create-resource.feature"))
```
Comment (askhat-abishev) [inline mod-linked-data/src/test/java/org/folio/CreateResourceSimulation.scala:33]: should ```create-resource.feature``` be changed to ```create-bib-api.feature```?

File: mod-linked-data/src/main/resources/citation/mod-linked-data/features/create-bib-api/create-bib-api.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Feature: Create Work and Instance resource using API & validate that they is created in mod-inventory and mod-search
```
Comment (askhat-abishev) [inline mod-linked-data/src/main/resources/citation/mod-linked-data/features/create-bib-api/create-bib-api.feature:1]: minor
...validate that they **are** created...
same on line 9

**Review by askhat-abishev — APPROVED** [review]

**Review by PBobylev — APPROVED** [review]

File: mod-linked-data/src/main/resources/citation/mod-linked-data/features/create-bib-api/create-bib-api.feature (hunk 3 above)
Comment (pkjacob) [reply mod-linked-data/src/main/resources/citation/mod-linked-data/features/create-bib-api/create-bib-api.feature:1]: Done

File: mod-linked-data/src/main/resources/karate-config.js [hunk 4]
```diff
@@ -97,13 +97,6 @@ function fn() {
     }
     config.prototypeTenant = '${prototypeTenant}';
     karate.configure('ssl',true);
-  } else if (env == 'ecs') {
```
Comment (pkjacob) [inline mod-linked-data/src/main/resources/karate-config.js:100]: Removing these to fix karate test failure in jenkins:

java.lang.Exception: Caught exception: groovy.lang.GroovyRuntimeException: Failed to parse template script (your template may contain an error or be trying to use expressions not currently supported): startup failed:
SimpleTemplateScript981.groovy: 105: illegal string body character after dollar sign;
   solution: either escape a literal dollar sign "\$5" or bracket the value expression "${5}" @ line 105, column 29.
         password: 'LinkedData$01'

**Pull request merged**